easybuild/easyconfigs/h/hanythingondemand/hanythingondemand-3.0.4-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'hanythingondemand'
+version = '3.0.4'
+
+homepage = 'https://github.com/hpcugent/hanythingondemand'
+description = """HanythingOnDemand (HOD) is a system for provisioning virtual Hadoop clusters over a large physical cluster.
+It uses the Torque resource manager to do node allocation."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+python = 'Python'
+pythonver = '2.7.10'
```

Comment:
we should probably wait with Python 2.7.11 until we have `intel/2016a` in place

this also relates to the dists we include, since they need to use the same Python version(?)